python_modules/dagster/dagster/_core/storage/tags.py

```diff
@@ -112,6 +112,8 @@
 
 POOL_TAG_PREFIX = f"{HIDDEN_TAG_PREFIX}pool/"
 
+EXTERNAL_JOB_SOURCE_TAG_KEY = f"{SYSTEM_TAG_PREFIX}/external-job-source"
```

Comment:
i hate hyphens and can this be `external_job_kind` (external_job_source is ok too just kinda like using `kind`)

Reply:
okie. I see we use underscores everywhere else so that's fine by me